ramses_inspector.py

```diff
@@ -22,28 +23,30 @@ def __init__(self,
                  scene: ExportableScene):
 
         self.scene = scene
-        self._window = None
+        self.viewer_process = None
 
-    def load_viewer(self):
+    def load_viewer(self, platform, block: bool = False):
         """Loads the RAMSES scene viewer for visual inspection of the
         generated scene"""
 
+        assert isinstance(platform, str)
+        platform = platform.lower()
+
         self.close_viewer()
 
         resolution_x = self.scene.blender_scene.render.resolution_x
         resolution_y = self.scene.blender_scene.render.resolution_y
 
-        self._window = self.scene.ramses.openWindow(resolution_x, resolution_y, 0, 0)
-        self._window.showScene(self.scene.ramses_scene)
+        program_args = f'-s {self.scene.output_path}{self.scene.blender_scene.name} -x -w {resolution_x} -h {resolution_y}'
```

Comment:
should adapt cmd line arguments to:
-x screenshot.png -xw ... -xh ...

Reply:
Fixed in 50414f9